common/src/main/java/com/wynntils/screens/maps/PoiCreationScreen.java

```diff
@@ -374,9 +377,8 @@ public void doRender(PoseStack poseStack, int mouseX, int mouseY, float partialT
 
         RenderUtils.disableScissor();
 
-        renderBackground(poseStack);
-        super.doRender(poseStack, mouseX, mouseY, partialTick);
-        // RenderUtils.renderDebugGrid(poseStack, GRID_DIVISIONS, dividedWidth, dividedHeight);
```

Comment:
You don't think we should keep the debug grid lines? I think they amount to some level of effort to getting them right. (In fact, we should maybe have them not as commented-out code but disabled by a global variable only.)

Reply:
I just removed the comment. The method is a renderutils one, can be called easily. This used to be a local method, hence the commented code.